flask_dynamo/manager.py

```diff
@@ -42,8 +42,10 @@ def wait_not_exists(self, table_name):
         self._wait(table_name, 'table_not_exists')
 
     def create_all(self, wait=False):
+        tables_name_list = [table.name for table in self._connection.tables.all()]
         for table in self._table_config:
-            self._connection.create_table(**table)
+            if table['TableName'] not in tables_name_list:
+                self._connection.create_table(**table)
         if wait:
```

Comment:
I think it's more efficient to ask all tables creation first, then wait for them to be created. Dynamo will manage in parallel all those creations and release our waiting step more quickly.